docs/toc.html

```diff
@@ -206,6 +206,14 @@
                     <li><a href="#connect_plugindiscovery">Plugin Discovery</a></li>
                 </ul>
                 <li><a href="#connect_development">8.3 Connector Development Guide</a></li>
+                <ul>
+                    <li><a href="#connect_concepts">Core Concepts and APIs</a></li>
+                    <li><a href="#connect_developing">Developing a Simple Connector</a></li>
+                    <li><a href="#connect_dynamicio">Dynamic Input/Output Streams</a></li>
+                    <li><a href="#connect_configs">Connect Configuration Validation</a></li>
```

Comment:
This is getting split across two lines when it's rendered. As a quick fix, maybe we can rename this section to just `Configuration Validation`?

Reply:
Yeah, it's definitely confusing - in fact I initially thought that I had messed it up somehow. I elected to keep it as is when I noticed that there are other headings with the same issue too (in other sections) but `Configuration Validation` sounds good to me so I've made that change.